Makefile

```diff
@@ -0,0 +1,64 @@
+# Set default shell to bash
+SHELL := /bin/bash -o pipefail -o errexit -o nounset
+
+addlicense=go run github.com/google/addlicense@v1.0.0 -ignore '**/*.yml' \
+	-ignore node_modules/*
+
+terramate_lsp_version=i4k-skeleton
+terramate_lsp_url=github.com/mineiros-io/terramate-lsp/cmd/terramate-lsp@$(terramate_lsp_version)
+go_install_dir=$(shell go env GOPATH)/bin
+
+.PHONY: default
+default: help
+
+## install deps
+.PHONY: deps
+deps:
+	npm install
+	go install "$(terramate_lsp_url)"
```

Comment:
Hmm to not depend on installing on host + knowing where it is installed + copying we could set [GOBIN=(https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies) to `./bin` and then just run go install, it should place the binary inside ./bin directly in that case.

Reply:
much better, thanks